### PR TITLE
[Release] Update dependencies

### PR DESCRIPTION
**Pull request by emlynmac**
# What
Bump stable chat dependency to current release version
Enable node 20 tooling

# Why
rush-managed version was set but not the package version.

# How Tested
CI

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: samples/StaticHtmlComposites/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "@azure/communication-react": "1.7.0-beta.1",
     "@azure/communication-common": "3.0.0-beta.1 || ^2.2.1",
     "@azure/communication-calling": "1.15.1-beta.1 || >=1.14.1",
-    "@azure/communication-chat": "1.3.2-beta.3 || >=1.2.0",
+    "@azure/communication-chat": "1.3.2-beta.3 || >=1.3.1",
```
Comment (vhuseinova-msft) [inline samples/StaticHtmlComposites/package.json:28]: It looks like 1.2.0 is required for stable version. Was this changed?
https://www.npmjs.com/package/@azure/communication-react?activeTab=code
image

Comment (emlynmac) [reply samples/StaticHtmlComposites/package.json:28]: The rush install is using 1.3.1 for the common versions
The range here just changes this to ensure it's at least 1.3.1 in the package.

**Review by JamesBurnside — CHANGES_REQUESTED** [review]
Also update the two `common-versions.json` files (we need to document this somewhere...)

**Comment by emlynmac** [comment]
> 

The stable one is already at 1.3.1

**Review by vhuseinova-msft — APPROVED** [review]

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**